### ISSUE TITLE
Error while running the server

### ISSUE THREAD

**Issue by bouhaddiamine** [Corvia/django-tenant-users#72]
After following all the steps twice : 
`django.core.management.base.SystemCheckError: SystemCheckError: System check identified some issues:

ERRORS:
permissions.UserTenantPermissions.groups: (fields.E304) Reverse accessor for 'UserTenantPermissions.groups' clashes with reverse accessor for 'CustomUser.groups'.
        HINT: Add or change a related_name argument to the definition for 'UserTenantPermissions.groups' or 'CustomUser.groups'.
permissions.UserTenantPermissions.user_permissions: (fields.E304) Reverse accessor for 'UserTenantPermissions.user_permissions' clashes with reverse accessor for 'CustomUser.user_permissions'.
        HINT: Add or change a related_name argument to the definition for 'UserTenantPermissions.user_permissions' or 'CustomUser.user_permissions'.
users.CustomUser.groups: (fields.E304) Reverse accessor for 'CustomUser.groups' clashes with reverse accessor for 'UserTenantPermissions.groups'.
        HINT: Add or change a related_name argument to the definition for 'CustomUser.groups' or 'UserTenantPermissions.groups'.
users.CustomUser.user_permissions: (fields.E304) Reverse accessor for 'CustomUser.user_permissions' clashes with reverse accessor for 'UserTenantPermissions.user_permissions'.
        HINT: Add or change a related_name argument to the definition for 'CustomUser.user_permissions' or 'UserTenantPermissions.user_permissions'.`

**Comment by hmamman** [comment]
@bouhaddiamine What steps did you follow, please?

**Comment by bouhaddiamine** [comment]
@hmamman   i followed the documentation  (https://github.com/Corvia/django-tenant-users)  / also i have django-tenants already installer

**Comment by bouhaddiamine** [comment]
@hmamman i need to let multiple users access to their tenants using the same domain ( but differents tenants) 

Ex  :     company.domain.com    so  each employe of the company( user ) can access his schema using the name of the company like that company.domain.com

**Comment by bouhaddiamine** [comment]
@hmamman i'm not sute if django-tenant-users is the solution or no, but i tried to install it and i got an error

**Comment by hmamman** [comment]
@bouhaddiamine are your users going to have access to multiple tenants at a? This is the main reason for using Django tenant users. It allows you to have all your users centralised at the public tenant in addition to enabling them belong to more than one tenant. 

If this is not how your application behaves, then Django tenant users is not for you, I guess.

**Comment by bouhaddiamine** [comment]
i don't know really because this is my first experience on multi tenancy,

i'm working on a CRM,   users(tenants) can for example share contacts
between them,   this is a case .
Do you think i need to use django-tenant-users in my case.
and also i would like to manage the permissions for example the company
owner ( master tenant ) can make / edit permission for the employees




Le lun. 3 mai 2021 à 13:44, Hussaini Mamman ***@***.***> a
écrit :

**Comment by bouhaddiamine** [comment]
The whole scenario of the application 

company create an account 
    - company can have multiple users 
    - the users can share the contacts between them 
    - all the company users must login using companyName.domain.com  for example 
    -

**Comment by bouhaddiamine** [comment]
my Big need is to have a fixed login page , after that i redirect the user
to his path if necessary

**Comment by bouhaddiamine** [comment]
my Big need is to have a fixed login page , after that i redirect the user
to his path if necessary
because i'm using DRF / VueJS,  that's why i need fixed login page

**Comment by hmamman** [comment]
@bouhaddiamine I don't think you need Django tenant users. You can be fine using Django tenants or Django tenant schema. You can then have a table at the public tenant for data sharing between users. Hope this will solve your issue.

**Comment by bouhaddiamine** [comment]
that's what i did but the issue now is login ( i'm using vuejs )  i need a
fixed path

Le lun. 3 mai 2021 à 14:01, Hussaini Mamman ***@***.***> a
écrit :

**Comment by hmamman** [comment]
@bouhaddiamine Django Tenant Users will be easier for your case, then.

**Comment by bouhaddiamine** [comment]
Must i do any migration durring the configuration of django tenant users ?
Because i get an error :(

**Comment by bouhaddiamine** [comment]
@hmamman  Must i do any migration durring the configuration of django tenant users ?
Because i get an error :(

**Comment by bouhaddiamine** [comment]
what is the the login end point for DRF with django tenants ?

**Comment by Dresdn** [comment]
@bouhaddiamine - Regarding your original error, do you have `AUTH_USER_MODEL` defined in your `settings.py`?
